docs/userGuide/usingPlugins.md

```diff
@@ -143,6 +143,34 @@ This will add the following link and script elements to the page:
 - `<script src="SCRIPT_LINK"></script>`
 - `<script>alert("hello")</script>`
 
+#### Live reload
+
+By default, MarkBind treats '.html', '.md', '.mbd', and '.mbdf' as source files, and will rebuild any
+pages changed when serving the page.
+
+During the `preRender` and `postRender` stages however, plugins may do custom processing using some other
+source file types, as parsed from the raw Markdown, typically requiring rebuilding the site.
+
+Hence, to add custom source files to watch, MarkBind allows defining an additional method in the plugin.
+- `getSources(content, pluginContext, frontMatter, config)`: Called before the `preRender` function to retrieve an array of source file paths to watch.
```

Comment:
**`Returns an array of source file paths to watch. `**`Called before a `**`Markdown file's`**`preRender function`**` is called.`**